en/orm/entities.rst

```diff
@@ -180,6 +180,57 @@ name will be the lower case and underscored version of the method::
 
 Do bear in mind that virtual fields cannot be used in finds.
 
+Storing explicit NULL values
+----------------------------
+
+By default CakePHP will store empty values such as empty strings as they come in.
```

Comment:
We currently do have issues here with bool as well, see my findings in the Core ticket.